lms/services/application_instance.py

```diff
@@ -154,6 +155,10 @@ def search(
                 tool_consumer_instance_guid=tool_consumer_instance_guid,
                 email=email,
             )
+            .order_by(
+                ApplicationInstance.last_launched.desc().nulls_last(),
+                ApplicationInstance.updated.desc(),
+            )
```

Comment:
I initially made this an optional argument, but there's no very neat way to accept a single value / list / None. You can't call `order_by` with a list and also we only want it ordered this one way for now.

If / when we see a different sort order, we can try and work out the best thing to do then.

Reply:
Seems like a sensible default for all searches for now :+1: